template/en/default/account/prefs/mfa.html.tmpl

```diff
@@ -16,10 +23,17 @@
   [% RETURN %]
 [% END %]
 [% IF user.cryptpassword == '*' %]
-  <input type="hidden" name="mfa_action" id="mfa-action" value="">
+  <input type="hidden" name="mfa_action" id="mfa-action" value="" data-nopassword="true">
+  <p>
+    Your account requires MFA to be configured before you can use [% terms.Bugzilla %]
+  </p>
+  <p>
+    As part of this transition you will no longer be able to use GitHub to log in to [% terms.Bugzilla %];
+    instead you will need to set a password for your existing account on Bugzilla and use that to login in future.
```

Comment:
You will no longer be able to use GitHub to log in to [% terms.Bugzilla %]; instead you will need to set a password for your existing account on Bugzilla and use that to log in with your MFA application.

The MFA provider applications which work with [% terms.Bugzilla %] are: Duo, Google, Blah, Blah, Blah. 

The Mozilla wiki has <a href="https://wiki.mozilla.com/path/to/page">instructions on enabling your 2FA application</a>.

Reply:
That wording isn't needed. Once they change the password the regular 2FA instructions will be shown.

Reply:
But do need to s/MFA/Two-Factor Authentication/